GithubHandler.py

```diff
@@ -144,11 +159,27 @@ def get_pr_deltas(self, pr):
         for f in file_changes:
             file_full_content = self.get_file_contents(f.filename, ref=pr.head.sha)
             full_file_delta = self.get_full_file_delta(f.filename, file_full_content)
-            file_delta = self.get_file_delta(f.patch, f.filename)
-            files_and_deltas += full_file_delta + "\n" + file_delta + "\n"
+            file_delta_str = self.get_file_delta(f.patch, f.filename)
+            files_and_deltas += full_file_delta + "\n" + file_delta_str + "\n"
 
         return files_and_deltas
 
+    def get_pr_deltas_in_list(self, pr):
```

Comment:
Hold my coffee... You've got this `get_pr_deltas_in_list` method, but it looks like it's turned up outta nowhere. Are you using this magic trick somewhere, or is it just for the showcase? Let's not bloat our codebase with abandoned artifacts.